theta_divide_num: 64
xy_divide_step: 0.01 # [m]

cost_scale: 1.0e3
cost_theta_scale: 0 # [m/rad]
step_cost: 1.0

heuristic_type: DijkstraPath
dijkstra_path_heuristic_expand_scale: 5.0

nominal_foot_separation: 0.2 # [m]

r2l_action_cont_list: # [m], [rad]
  [
  [0.2, 0, 0],
  [-0.1, 0, 0],
  [0, 0.1, 0],
  [0, -0.05, 0],
  [0, 0, 0.349],
  [0, 0, -0.349]
  ]

r2l_reachable_min: [-0.05, 0.0, -0.1745] # [m], [rad]
r2l_reachable_max: [0.1, 0.05, 0.1745] # [m], [rad]

# [center_x, center_y, half_length_x, half_length_y]
rect_obstacle_list:
  [
    [-2.5, 0.0, 0.2, 1.0],  # 왼쪽 벽
    [2.5, 0.0, 0.2, 2.5],   # 오른쪽 벽
    [0.0, -2.5, 2.5, 0.2],  # 위쪽 벽
    [0.0, 2.5, 2.5, 0.2],   # 아래쪽 벽
    
    [-4, 0.0, 0.2, 4],  # 왼쪽 벽
    
  ]
